core/build.gradle

```diff
@@ -0,0 +1,11 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    mavenLocal()
+}
+
+dependencies {
+    implementation 'com.epam:utils:1.3.5'
+}
```

Comment:
Ancient systems were not able to process the last line if no newline symbol was provided. Nowadays I believe you won't face with such problems but it's a good practice to add a newline to the end of file.